tests/rules/just.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { stripIndent } from "common-tags";
+import rule = require("../../source/rules/just");
+import { ruleTester } from "../utils";
+
+ruleTester({ types: true }).run("just", rule, {
+  valid: [
+    stripIndent`
+      function foo(): void {
+        function of(): void {}
+        of();
+      }
+
+      function bar(of: Function): void {
+          of();
+      }
+
+      function baz(): void {
+          const of = () => {};
+          of();
+      }
+    `,
+    stripIndent`
+      import { of as bar } from "rxjs";
+
+      const a = bar("a");
+    `
+  ],
+  invalid: [
+    {
+      code: stripIndent`
+        import { of } from "rxjs";
+
+        const a = of("a");
+      `,
+      output: stripIndent`
+        import { of as just } from "rxjs";
+
+        const a = just("a");
+      `,
+      errors: [
+        {
+          messageId: "forbidden",
+          line: 1,
+          column: 10,
+          endLine: 1,
+          endColumn: 12
+        },
+        {
+          messageId: "forbidden",
+          line: 3,
+          column: 11,
+          endLine: 3,
+          endColumn: 13
+        }
+      ]
+    },
+    {
+      code: stripIndent`
+        import { of as of } from "rxjs";
```

Comment:
Should this be valid? 
(It is valid in TSLint)

Reply:
It's valid JS, so yes it should be valid. But if it makes the implementation way more painful, just ignore it. I mean, who is going to write that? 🙈

Reply:
Seems like I just had to open my eyes... `import { of } from "rxjs"` has also a local identifier with the name `of` (that's why I thought it would be hard to implement). But, the node has the same `loc` and `range` as the imported identifier.

Short said, we can assume that if the `range` from the local identifier is different from the import identifier, we're dealing with a renamed import.